budgetportal/prov_infra_project/charts.py

```diff
@@ -0,0 +1,131 @@
+
+def order_chart_data(snapshot_list):
+    return sorted(snapshot_list, key=lambda x: x["date"])
+
+
+def time_series_data(project_snapshots):
```

Comment:
Let's just document that this expects a collection of ProvInfraProjectSnapshots of the same project, in IRMSnapshot order.

Reply:
Docstring added

https://github.com/vulekamali/datamanager/pull/215/commits/821d94b299019a7985f9ac13a832e99b691b10f0